api/tests/Feature/PoolTest.php

```diff
@@ -1118,6 +1118,8 @@ public function testPoolNameScope(): void
      */
     public function testPoolTeamScope(): void
     {
+        $this->markTestSkipped('PoolFilterInput does not have community argument.');
```

Comment:
This was out of scope so I decided to skip the test instead of removing the test altogether as filtering a pool by community is likely a desired feature.

Reply:
- https://github.com/GCTC-NTGC/gc-digital-talent/issues/12586